docs/BLOC_NAMING_CONVENTIONS.md

```
@@ -0,0 +1,82 @@
+
+`initial` | `success` | `failure` | `loading`.
+
+:::note
+The base state class should always be named: `BlocSubject` + `State`.
+:::
+
+### Examples
+
+✅ **Good**
+
+##### Subclasses
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h4; Actual: h5

(MD001, heading-increment)